pkg/plugins/generic/generic_plugin.go

```diff
@@ -170,6 +170,12 @@ func tryEnableIommuInKernelArgs() (bool, error) {
 	cmd.Stderr = &stderr
 
 	if err := cmd.Run(); err != nil {
+		// if grubby is not there log and assume kernel args are set correctly.
+		if exiterr, ok := err.(*exec.ExitError); ok && exiterr.Error() == "exit status 127" {
```

Comment:
I think it would be better to check the exit status with
```
        if status, ok := exitErr.Sys().(syscall.WaitStatus); ok &&  status.ExitStatus() == 127 {
            ...
        }
```

Reply:
Thanks!